examples/sri-1/components/workgroup/workgroup/workgroup.ts

```diff
@@ -0,0 +1,112 @@
+import { Workstep } from '../workstep/workstep';
+import { BpiSubject } from '../identity/bpiSubject';
+import { IWorkgroup } from './workgroup.interface';
+import { Workflow } from '../workflow/workflow';
+import { Security } from '../policy/security'
+import { Privacy } from "../policy/privacy"
+
+export class Workgroup implements IWorkgroup {
+    id: string;
+    name: string;
+    administrator: BpiSubject[] = []; //[R238] A workgroup MUST have at least one administrator.
+    securityPolicy: Security[] = []; //[R236] There MUST be at least one BPI Subject role that has the authorization to create a workgroup. //[239] A workgroup MUST have at least one security policy. Note that a security policy consists of authentication and authorization rules for the workgroup participants. Note also that one or more workgroup administrators define the workgroup security policy. //TODO Implement securityPolicy #485
+    privacyPolicy: Privacy[] = []; //[R240] A workgroup MUST have at least one security policy. A privacy policy consists of the data visibility rules for each participant. //TODO Implement privacyPolicy #485
+    participants: BpiSubject[] = [];
+    worksteps: Workstep[] = [];
+    workflows: Workflow[] = [];
+
+    constructor(id: string, name: string, administrators: BpiSubject[], participants: BpiSubject[], securityPolicy: Security, privacyPolicy: Privacy ) {
+        this.id = id;
+        this.name = name;
+        this.administrator.push(administrators); //[R238] A workgroup MUST have at least one administrator.
+        this.participants.push(participants); //[R237] A workgroup MUST consist of at least one BPI Subject participant.
+        this.securityPolicy.push(securityPolicy); //[239] A workgroup MUST have at least one security policy. Note that a security policy consists of authentication and authorization rules for the workgroup participants. Note also that one or more workgroup administrators define the workgroup security policy.
+        this.privacyPolicy.push(privacyPolicy); //[R240] A workgroup MUST have at least one security policy. A privacy policy consists of the data visibility rules for each participant.
+    }
+
+    //TODO Implement checks for actions bound by security and privacy policies #485
+    addParticipant(bpiSubject: BpiSubject) { //[R241] A workgroup administrator MUST be able to perform at minimum the following functions
+        return  this.participants.push(bpiSubject);
```

Comment:
I would actually vote we ditch all this implementation as it is too early, and instead just throw not implemented. These behaviors will be implemented one by one as we setup the architecture and start introducing functionalities. For example, add participant might be added through a AddParticipantCommand that will have to do a number of checks before adding the participant of the workgroup to the DB.

Reply:
removed these stand-ins